angular/projects/spark-angular/src/lib/components/sprk-ordered-list/sprk-list.stories.ts

```diff
@@ -0,0 +1,106 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkOrderedListModule } from './sprk-ordered-list.module';
+import { SprkUnorderedListModule } from '../sprk-unordered-list/sprk-unordered-list.module';
+import { SprkListItemModule } from '../sprk-list-item/sprk-list-item.module';
+
+export default {
+  title: 'Components|List',
+  component: SprkUnorderedListModule,
+  decorators: [
+    storyWrapper(
+      storyContent => (
+        `<div class="sprk-o-Box sprk-u-mlm">${ storyContent }<div>`
```

Comment:
do we need this sprk-u-mlm?

Reply:
Yes, the unordered list appears like this without it... 
<img width="218" alt="Screen Shot 2019-10-31 at 1 27 39 PM" src="https://user-images.githubusercontent.com/15703773/67970937-3dd41280-fbe2-11e9-85fa-6b2dbcb8170c.png">

Reply:
@guern1kn how come the angular SB needed the margin left and the HTML one didn't?

Reply:
@ssabo1 Can we create a research issue to look into this? The angular component list items are rendering the text at the start of the padding, while the html is rendering the dot/number at the start of the padding. Not 100% sure if this is a bug or just how Angular works. 
![image](https://user-images.githubusercontent.com/15703773/68034647-70404700-fc98-11e9-9833-9c7901ad8dba.png)
![image](https://user-images.githubusercontent.com/15703773/68034672-7c2c0900-fc98-11e9-890e-f7ec824a813c.png)

@afebbraro